src/app/page/cabinet/cabinet.component.ts

```diff
@@ -24,7 +24,7 @@ export class CabinetComponent implements OnInit {
   public decks: DeckLinkByCategory[];
   public categoryDecks: DeckLinkByCategory[];
   public showCourseDecks: any;
```

Comment:
showCourseDecks should also be boolean?

Reply:
No, it shouldn't. It holds courseId sometimes.